api-reference/testrpc/methods.md

```diff
@@ -0,0 +1,151 @@
+# Echo TestRPC API
+
+### eth_mining
+Returns `true` if client is actively mining new blocks, otherwise `false`.
+
+##### Example
+```json
+// Request
+curl -X POST --data '{"jsonrpc":"2.0","method":"eth_mining","params":[],"id":1}'
+
+// Result
+{
+  "id":1,
+  "jsonrpc": "2.0",
+  "result": true
+}
+```
+
+### miner_start
+Start the block mining process.
+
+##### Example
+```json
+// Request
+curl -X POST --data '{"jsonrpc":"2.0","method":"miner_start","params":[],"id":1}'
+
+// Result
+{
+  "id":1,
+  "jsonrpc": "2.0",
+  "result": true
+}
+```
+
+### miner_stop
+Stop the block mining process.
+
+##### Example
+```json
+// Request
+curl -X POST --data '{"jsonrpc":"2.0","method":"miner_stop","params":[],"id":1}'
+
+// Result
+{
+  "id":1,
+  "jsonrpc": "2.0",
+  "result": true
+}
+```
+
+### personal_newAccount
+Generates a new private key and create new account with it.  
+
+| Option | Description |
+| :--- | :--- |
+| `string name` | New account name |
+| `string passphrase` | Passphrase which is used to unlock account |
+| Returns | The address of the new account |
+
+##### Example
+```json
+// Request
+curl -X POST --data '{"jsonrpc":"2.0","method":"miner_stop","params":["swordfish"],"id":1}'
+
+// Result
+{
+  "id":1,
+  "jsonrpc": "2.0",
+  "result": true
+}
+```
+
+### personal_listAccounts
+Returns all account addresses of all keys in the key store.
+
+### personal_lockAccounts
+Removes the private key with given address from memory. The account can no longer be used to send transactions.
+
+### personal_unlockAccounts
+Decrypts the key with the given address from the key store.  
+The account can be used with eth_sign and eth_sendTransaction while it is unlocked.
+
+### eth_sign
+The sign method calculates an Ethereum specific signature with: sign(keccak256("\x19Ethereum Signed Message:\n" + len(message) + message))).
+
+**Note** the address to sign with must be unlocked.
+
+| Option | Description |
```

Comment:
It seems some kind of custom data format is used here, not the ethereum format described here - https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_sign

And we cannot use account name as an argument, it's incompatible with ethereum. We should use 0x address only.